sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -35,6 +36,10 @@ class DictVectorizer(TransformerMixin, BaseEstimator):
     a feature "f" that can take on the values "ham" and "spam" will become two
     features in the output, one signifying "f=ham", the other "f=spam".
 
+    When feature values are iterables of strings (not mappings), this
```

Comment:
Maybe less precise here would be more helpful than the precise but uninstructive "iterables of strings (not mappings)". "If a feature value is a sequence or set of strings, this ..."